docs/source/tutorial/setup_analysis_project.rst

```diff
@@ -196,39 +196,61 @@ Leave this alone.
 Timeseries CSV Export
 ^^^^^^^^^^^^^^^^^^^^^
 
-If you do not need the timeseries data for your simulations, you can skip this measure to save disk space. Otherwise, one csv file per datapoint will be written containing timeseries data for their model. After `downloading all datapoints <run_project.html#download>`_ to your project's localResults folder, each datapoint's ``enduse_timeseries.csv`` file will be contained in a zipped ``data_point.zip`` file along with all other simulation input and output files.
+If you do not need the timeseries data for your simulations, you can skip this measure to save disk space. Otherwise, one csv file per datapoint will be written containing end use timeseries data for their model. After `downloading all datapoints <run_project.html#download>`_ to your project's localResults folder, each datapoint's ``enduse_timeseries.csv`` file will be contained in a zipped ``data_point.zip`` file along with all other simulation input and output files.
   
 .. image:: ../images/tutorial/timeseries_csv_export.png
 
+End uses include:
+
+  * heating [electric/gas/propane/oil] [kWh/kBtu/kBtu/kBtu]
+  * cooling [kWh]
+  * interior lights [kWh]
+  * exterior lights [kWh]
+  * interior equipment [electric/gas/propane/oil] [kWh/kBtu/kBtu/kBtu]
+  * fans [kWh]
+  * pumps [kWh]
+  * water heating [electric/gas/propane/oil] [kWh/kBtu/kBtu/kBtu]
+  * water [gal]
+  * pv [kWh]
+
 **Reporting Frequency**
-  The timeseries data will be reported at hourly intervals unless otherwise specified. Other options include:
+  The timeseries data will be reported at hourly intervals unless otherwise specified. Alternative reporting frequencies include:
 
-  * Detailed
-  * Timestep
-  * Daily
-  * Monthly
-  * RunPeriod
+  * detailed
+  * timestep
+  * daily
+  * monthly
+  * run period
   
-  Setting the reporting frequency to "Timestep" will give you interval output equal to the timestep set by the "Simulation Controls" measure. Thus by default, this measure will produce 10-min interval output.
+  Setting the reporting frequency to "detailed" will give you interval output equal to the HVAC system timestep. Setting the reporting frequency to "timestep" will give you interval output equal to the zone timestep set by the "Simulation Controls" measure. Thus, this measure will produce 10-min interval output when you select "timestep" and leave the "Simulation Controls" measure at its default settings.
```

Comment:
Doesn't "detailed" give some outputs at the HVAC system timestep and some at the zone timestep?

Reply:
Not totally sure. I know you can get two different intervals if you select "detailed", but not sure if the second is "zone timestep".

Reply:
From https://bigladdersoftware.com/epx/docs/9-0/input-output-reference/input-for-output.html#inputs-8-019: "Detailed” will list the value each calculation step (i.e. Zone or HVAC)

Reply:
Thanks for pointing this out.